ideascube/conf/kb_bfa_bsfcampus.py

```diff
@@ -0,0 +1,5 @@
+"""KoomBook conf"""
+from .kb_bsfcampus import *  # pragma: no flakes
+
+LANGUAGE_CODE = 'fr'
+IDEASCUBE_NAME = 'BSF Campus Burkina Faso'
```

Comment:
Do you really need the `IDEASCUBE_NAME` config?

We're trying to move away from it, in favour of the in-database configuration system:

```
$ ideascube config set server site-name 'BSF Campus Burkina Faso'
```